internal/pkg/fieldtags/testdata/src/tests/test.go

```diff
@@ -18,6 +18,7 @@ type Person struct {
 	password, creds      string      `levee:"source"`               // want "tagged field: password, creds"
 	secret               string      `json:"secret" levee:"source"` // want "tagged field: secret"
 	another              interface{} "levee:\"source\""             // want "tagged field: another"
+	hasCustomFieldTag    string      `example:"sensitive"`          // want "tagged field: hasCustomFieldTag"
```

Comment:
I don't know if we care, but this file currently fails `go vet -structtag`.  It's against convention / `reflect.StructTag` to tag unexported fields.

Reply:
Interesting. Do you have a link for that? I would be curious to read more. This convention sounds strange to me, I thought struct tags were intended for adding metadata in general.

While looking for a link I looked at `reflect.StructTag` and I think it might be a good idea to rely on that instead of duplicating the code. I don't remember why I didn't use that initially.

Reply:
Oh, I just ran it locally, and [the go vet docs](https://golang.org/cmd/vet/) claim

> structtag    check that struct field tags conform to reflect.StructTag.Get

I only bring up `StructTag` because it seems to have defined the community standard.